### ISSUE TITLE
Add an alternative merge mechanism for binary files

### ISSUE THREAD

**Issue by zoktiik** [AlanParadis/Stalker2SimpleModMerger#9]
Is it possible that, for the merging functionality, we could specify whether to treat some files or file extensions as binary, and when we do, it becomes a very simple either-or pick-and-choose within the CLI for which file to take? That would be much more clear and easier to work with. My current load order has a single conflict: a .wem (Wwise audio) file between two mods. But attempting to manual merge using Kdiff3 takes a LONG time to load, and you can close the window anyway for it to...somehow...just decide which file to take. I get why the merger is there: configuration files! But the merger has a process which assumes your conflicts will *only* be between conflicting configs and not between binary files. Auto merge just seems to be stuck forever if you attempt to use it in a situation like this. It's also important to note that Kdiff3 doesn't seem to have a way to **fully reject the other file automatically**.

There very well could be a better implementation than the one I thought of off the top of my head; the example I described at the top isn't super-necessary to follow.

And, as others have mentioned in the Nexus comments, a better explanation of what the merge methods and the merging process does would be nice to have as well.

**Comment by ChemGuy1611** [comment]
.wem are not binary files. They are sound files. It is not possible to "merge" those files. They are individual sounds, like a song in an mp3. This is like asking if you can merge two different songs into one by simply smashing the bits together. You have to use load ordering to select which mod will win the conflict for sound files.

**Comment by zoktiik** [comment]
Right, my terminology is wrong (I used "binary" to refer to non-text files), and I have a big wall of text which makes it confusing to read. 
The point I was trying to make is: BECAUSE you can't merge these file types, there needs to be the ability to take one or the other completely, and skip the "manual/auto merge" question. If you use option 3 (skip) it just doesn't make a merge patch at all. The workaround is to select manual merge and then close Kdiff3, which will just take one of the mod files (it's unclear which) and proceed with the production of the merged patch.

**Comment by ChemGuy1611** [comment]
This tool only merges .CFG files. It only puts cfg files in the merged patch pak. Nothing else.

It sounds like what you are saying is that your mods have cfg and wem files in a single pak.

For this you can use Vortex Mod Manager to solve this problem. It has load order functionality and will not remove the original mod paks from the ~mods folder. Or you will need to pull those paks out of the backup folder and rename them manually.
